doc/man7/migration_guide.pod

```diff
@@ -926,7 +926,8 @@ L<OSSL_LIB_CTX_free(3)>, L<OSSL_LIB_CTX_load_config(3)> and L<OSSL_LIB_CTX_set0_
 
 L<OSSL_PROVIDER_add_builtin(3)>, L<OSSL_PROVIDER_available(3)>,
 L<OSSL_PROVIDER_do_all(3)>, L<OSSL_PROVIDER_load(3)>,
-L<OSSL_PROVIDER_set_default_search_path(3)> and L<OSSL_PROVIDER_try_load(3)>
+L<OSSL_PROVIDER_set_default_search_path(3)>,
+L<OSSL_PROVIDER_get1_default_search_path(3)> and L<OSSL_PROVIDER_try_load(3)>
```

Comment:
This is currently a migration guide for 3.0
So unless we are going to merge this back to 3.0, it doesnt belong in this section I think.

Reply:
Ah, is the migration just for 3.0?
This is a getter, and looks like it _could_ be backported, but it doesn't sound like there is much desire to do so.
So shall I just remove this addition?

Reply:
Currently it is for 3.0 (See the section heading at the top of the file)..
Maybe do it as a CHANGES entry..
@openssl/otc Should we be adding new sections to the migration guide for 3.2?

Reply:
or should we back port the missing accessor?

Reply:
+1 to backport (at least to 3.1)

Reply:
Ok, removed migration docs.